src/main/java/cinema/config/AppConfig.java

```diff
@@ -14,7 +14,7 @@
 
 @Configuration
 @PropertySource("classpath:db.properties")
-@ComponentScan(basePackages = {"cinema"})
+@ComponentScan(basePackages = {"cinema.model"})
```

Comment:
why `model` here? it should be only for configuring entity list, here we should write the root directory for searching all beans that we have